.github/workflows/ci.yml

```diff
@@ -70,7 +70,7 @@ jobs:
     name: Build and deploy to Azure Container Registry
     runs-on: ubuntu-latest
     environment:
-      name: azure container registry
+      name: az container registry
```

Comment:
@iferencik please revert this change to use correct environment with registered secret.

![image](https://github.com/user-attachments/assets/6d831049-1eea-40cd-8b88-f9b209a7be0c)